clients/admin-ui/src/features/common/form/inputs.tsx

```diff
@@ -169,6 +169,7 @@ const SelectInput = ({
       name={fieldName}
       value={selected}
       size={size}
+      classNamePrefix="custom-select"
```

Comment:
react-select attaches this to its various components, which allows the tests to cleanly target specific elements.

Reply:
nice!